src/uct/ib/rc/verbs/rc_verbs_common.c

```diff
@@ -36,18 +36,18 @@ void uct_rc_verbs_iface_common_query(uct_rc_verbs_iface_common_t *verbs_iface,
     iface_attr->cap.put.max_short = verbs_iface->config.max_inline;
     iface_attr->cap.put.max_bcopy = iface->super.config.seg_size;
     iface_attr->cap.put.max_zcopy = uct_ib_iface_port_attr(&iface->super)->max_msg_sz;
-    iface_attr->cap.put.max_iov   = 1;
+    iface_attr->cap.put.max_iov   = iface->super.super.config.max_iov;
 
     /* GET */
     iface_attr->cap.get.max_bcopy = iface->super.config.seg_size;
     iface_attr->cap.get.max_zcopy = uct_ib_iface_port_attr(&iface->super)->max_msg_sz;
-    iface_attr->cap.get.max_iov   = 1;
+    iface_attr->cap.get.max_iov   = iface->super.super.config.max_iov;
 
     /* AM */
     iface_attr->cap.am.max_short  = verbs_iface->config.max_inline - sizeof(uct_rc_hdr_t);
     iface_attr->cap.am.max_bcopy  = iface->super.config.seg_size - sizeof(uct_rc_hdr_t);
     iface_attr->cap.am.max_zcopy  = iface->super.config.seg_size - sizeof(uct_rc_hdr_t);
-    iface_attr->cap.am.max_iov    = 1;
+    iface_attr->cap.am.max_iov    = iface->super.super.config.max_iov - 1;
```

Comment:
Can you please add short comment here and I will merge it. For such minor change I will not wait for full regression cycle.